### PR TITLE
Get token from context instead of .env

### PR DESCRIPTION
**Pull request by Sphinxs**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "react-chartjs-2": "2.8.0",
     "react-copy-to-clipboard": "5.0.1",
     "react-datetime": "2.16.3",
-    "react-dom": "16.8.4",
+    "react-dom": "16.10.2",
```
Comment (iguit0) [inline package.json]: What's the reason to update?
Old version crashes while running tests?

Comment (Sphinxs) [reply package.json]: Yep, at least here, this had add new warnings in the browser console, warnings to update the way a few things are done

Comment (Sphinxs) [reply package.json]: @iguit0 the erros where fixed, please review the merge again.

**Comment by Sphinxs** [comment]
@iguit0 the erros where fixed, please review the merge again.

**Comment by iguit0** [comment]
> @iguit0 the erros where fixed, please review the merge again.

Works smoothly now :smiley: :tada:

**Pull request merged**